<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PluginPanel</class>
 <widget class="QWidget" name="PluginPanel">
  <property name="windowTitle">
   <string>Plugin Panel</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">

   <!-- Top controls: plugin directories + rescan -->
   <item>
    <layout class="QHBoxLayout" name="dirsLayout">
     <item>
      <widget class="QLabel" name="dirsLabel">
       <property name="text">
        <string>Plugin directories (':' or ';' separated):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="dirsEdit">
       <property name="placeholderText">
        <string>./plugins</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="rescanButton">
       <property name="text">
        <string>Rescan</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>

   <!-- Table of plugins -->
   <item>
    <widget class="QTableWidget" name="pluginsTable">
     <property name="columnCount">
      <number>6</number>
     </property>
     <column>
      <property name="text">
       <string>Active</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Kind</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Path</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Description</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Version</string>
      </property>
     </column>
    </widget>
   </item>

   <!-- Bottom buttons -->
   <item>
    <layout class="QHBoxLayout" name="buttonsLayout">
     <item>
      <widget class="QPushButton" name="activateAllButton">
       <property name="text">
        <string>Activate All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deactivateAllButton">
       <property name="text">
        <string>Deactivate All</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>

  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
